### DISCUSSION TITLE
关于备案号填写的问题

### DISCUSSION THREAD

**Discussion by SkyWave2022** [bs-community/blessing-skin-server#398] [Q&A] [answered]
作者可以添加填写备案号的入口吗？ 
![image](https://user-images.githubusercontent.com/86274592/164147876-75932568-7212-49b0-83be-6ed465e7ac45.png)

**Comment by Big-Cake-jpg** [comment, marked as answer]
BS 已经提供了自定义页脚的功能，只要会一点点 HTML 就行了

从 Windows 版邮件发送

发件人: SkyWave
发送时间: 2022年4月20日 12:02
收件人: bs-community/blessing-skin-server
抄送: Subscribed
主题: [bs-community/blessing-skin-server] 关于备案号填写的问题 (Discussion #398)

作者可以添加填写备案号的入口吗？

**Reply by Big-Cake-jpg** [reply]
![image](https://user-images.githubusercontent.com/76810494/164343310-34fe26af-d0e7-4d0a-9bb8-62dce5fb5c11.png)

**Comment by SkyWave2022** [comment]
谢谢，是我眼瞎了
